photutils/02_photutils_source_detection.ipynb

```diff
@@ -0,0 +1,1034 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up the figure with subplots\n",
+    "fig, ax1 = plt.subplots(1, 1, figsize=(8, 8))\n",
+    "\n",
+    "# Plot the data\n",
+    "fitsplot = ax1.imshow(np.ma.masked_where(xdf_image.mask, xdf_image_clipped), norm=norm_image)\n",
+    "ax1.scatter(sources_findpeaks['x_peak'], sources_findpeaks['y_peak'], s=30, marker='o', \n",
+    "            lw=1, alpha=0.7, facecolor='None', edgecolor='r')\n",
+    "\n",
+    "# Define the colorbar\n",
+    "cbar = plt.colorbar(fitsplot, fraction=0.046, pad=0.04, ticks=LogLocator(subs=range(10)))\n",
+    "labels = ['$10^{-4}$'] + [''] * 8 + ['$10^{-3}$'] + [''] * 8 + ['$10^{-2}$']\n",
+    "cbar.ax.set_yticklabels(labels)\n",
+    "\n",
+    "# Define labels\n",
+    "cbar.set_label(r'Flux Count Rate ($e^{-1}/s$)', rotation=270, labelpad=30)\n",
+    "ax1.set_xlabel('X (pixels)')\n",
+    "ax1.set_ylabel('Y (pixels)')\n",
+    "ax1.set_title('find\\_peaks Sources')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And a closer look:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig, axs = plt.subplots(3,3, figsize=(3, 3))\n",
+    "plt.subplots_adjust(wspace=0.1, hspace=0.1)\n",
+    "\n",
+    "cutout_size = 20\n",
+    "\n",
+    "srcs = np.random.permutation(sources_findpeaks)[:axs.size]\n",
+    "for ax, src in zip(axs.ravel(), srcs):\n",
+    "    slc = (slice(int(src['y_peak'] - cutout_size), int(src['y_peak'] + cutout_size)),\n",
+    "           slice(int(src['x_peak'] - cutout_size), int(src['x_peak'] + cutout_size)))\n",
+    "    ax.imshow(xdf_image_clipped[slc], norm=norm_image)\n",
+    "    src_id = np.where((sources_findpeaks['x_peak'] == src['x_peak']) & \n",
+    "                      (sources_findpeaks['y_peak'] == src['y_peak']))[0][0]\n",
+    "    ax.text(2, 2, str(src_id), color='w', va='top')\n",
+    "    ax.set_xticks([])\n",
+    "    ax.set_yticks([])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Comparing Detection Methods\n",
+    "\n",
+    "Let's compare how each of these different strategies did.\n",
+    "\n",
+    "First, how many sources did each method find?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('''DAOStarFinder: {} sources\n",
+    "IRAFStarFinder: {} sources\n",
+    "find_peaks: {} sources'''.format(len(sources_dao), len(sources_iraf), len(sources_findpeaks)))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, how many of these sources match? We can answer this question by using [sets](https://docs.python.org/2/library/sets.html) to compare the centroids of the different sources (rounding to the first decimal place)."
```

Comment:
this seems to link to the python 2 version of the docs.  I think you want just the version-less one?: https://docs.python.org/library/stdtypes.html#set-types-set-frozenset